lib/components/primitive-components/Group/GroupInterfacePort.ts

```diff
@@ -0,0 +1,42 @@
+import type { Port as ChildPort, PortProps } from "../Port/Port"
+import { Port } from "../Port/Port"
+
+export class GroupInterfacePort extends Port {
+  private _targetPorts: ChildPort[]
+  private _targetSelectors: string[]
+
+  constructor(
+    props: PortProps,
+    opts: { targetPorts: ChildPort[]; targetSelectors: string[] },
+  ) {
+    super(props)
+    this._ignoreParentSchematicSuppression = true
+    this._targetPorts = opts.targetPorts
+    this._targetSelectors = opts.targetSelectors
+  }
+
+  get targetPorts(): ChildPort[] {
+    return this._targetPorts
+  }
+
+  get targetSelectors(): string[] {
+    return this._targetSelectors
+  }
+
+  doInitialSourceRender(): void {
+    const primaryTarget = this._targetPorts[0]
+    if (!primaryTarget?.source_port_id) return
+    this.source_port_id = primaryTarget.source_port_id
+    this.source_component_id = primaryTarget.source_component_id
+    this.subcircuit_connectivity_map_key =
+      primaryTarget.subcircuit_connectivity_map_key ?? null
+  }
+
+  doInitialSourceParentAttachment(): void {
+    // The underlying target port already has the correct parent attachment.
```

Comment:
...then why are you overriding this method? by overriding this method you are removing the implementation for source parent attachment